src/main/java/org/commcare/suite/model/QueryPrompt.java

```diff
@@ -62,6 +64,7 @@ public QueryPrompt(String key, String appearance, String input, String receive,
         this.display = display;
         this.itemsetBinding = itemsetBinding;
         this.defaultValueExpr = defaultValueExpr;
+        this.exclude = exclude;
```

Comment:
this needs to be added as a parameter to the constructor

Reply:
Thanks, missed that when transferring changes from testing in formplayer.